tools/doc/README.md

```diff
@@ -98,9 +101,9 @@ This event is emitted on instances of `SomeClass`, not on the module itself.
 ```
 
 
-* Classes have (description, Properties, Methods, Events).
```

Comment:
Is there a reason behind this reordering?
The existing one is sorted in alphabetical order.

Reply:
I mean some logical hierarchy: from parent structures to child components mentioned in the previous lines. If this is not clear, I can revert this fragment,

Reply:
Classes and Functions go together, in most languages classes contain functions but in JavaScript everything is a function at the end.
Alphabetical order might be better.

Reply:
Reverted)